src/entities/supportRequest.ts

```diff
@@ -10,11 +10,13 @@ export enum SupportRequestErrors {
 
 @Entity()
 export class SupportRequest extends BaseEntity {
-  constructor(slackIdId: string, name: string, type: SupportRequestType) {
+  constructor(slackIdId: string, name: string, type: SupportRequestType, primaryLanguage = '', problemDescription = '') {
     super();
 
     this.slackId = slackIdId;
     this.name = name;
+    this.primaryLanguage = primaryLanguage;
+    this.extraData = { problemDescription };
```

Comment:
Look at the other change talking about the interface, but this is where you would use that.

```suggestion
  constructor(slackIdId: string, name: string, type: SupportRequestType, primaryLanguage = '', extraData: SupportRequestExtraData = { problemDescription: '' }) {
    super();

    this.slackId = slackIdId;
    this.name = name;
    this.primaryLanguage = primaryLanguage;
    this.extraData = extraData;
```